tests/wea_test.py

```diff
@@ -136,6 +136,17 @@ def test_import_stat(self):
         assert wea_from_json.diffuse_horizontal_radiation.values == \
             wea_from_stat.diffuse_horizontal_radiation.values
 
+    def test_write_wea(self):
+        """Test the write Wea file capability."""
+        stat_path = './tests/stat/chicago.stat'
+        wea_from_stat = Wea.from_stat_file(stat_path)
+
+        wea_path = './tests/wea/chicago_stat.wea'
+        hoys = range(8760)
+        wea_from_stat.write(wea_path, hoys, True)
+
+        assert os.path.isfile(wea_path)
```

Comment:
Again I would check the values in the wea file and the header in addition to if the file has been created.

Reply:
Done!